en/setup/_gcc_toolchain_installation.txt

```diff
@@ -0,0 +1,68 @@
+<!-- GCC toolchain documentation used for all Linux platforms to build NuttX -->
```

Comment:
FYI this is common "additional tools" text that is currently pulled into NuttX and Arch/Cent OS docs. May not be needed in the other linux topics (raised as a question later).